pkg/destroy/aws/shared.go

```diff
@@ -35,13 +35,14 @@ func (o *ClusterUninstaller) clusterOwnedKeys() []string {
 	var keys []string
 	for _, filter := range o.Filters {
 		for key, value := range filter {
-			if !strings.HasPrefix(key, "kubernetes.io/cluster/") {
-				continue
-			}
-			if value != "owned" {
-				o.Logger.Warnf("Ignoring non-owned cluster key %s: %s for shared-tag removal", key, value)
```

Comment:
Although this warning message says that a value other than "owned" would be ignored, it actually doesn't. So, this fix matches the implementation with what the warning message says. Hopefully that is the intention of this method also.

Reply:
Not pursuing this portion of the fix anymore.